var/spack/repos/builtin/packages/bambam/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Bambam(MakefilePackage):
+    """Bambam is a tool used to facilitate NGS analysis."""
+
+    homepage = "http://udall-lab.byu.edu/Research/Software/BamBam"
+    url      = "https://downloads.sourceforge.net/project/bambam/bambam-1.4.tgz"
+
+    version('1.4', sha256='a9b178251d771aafb8c676d30a9af88ea2fc679c2a5672b515f86be0e69238f1')
+    version('1.3', sha256='de91af03c1c09921a3a7f816b4df0564213e57c1a6024f2bcd63c0e8f0733a50', preferred=True)
+
+    depends_on('perl', type=('build', 'run'))
+    depends_on('samtools')
+    depends_on('bamtools@2.3.0')
```

Comment:
are you restricted to this special version?

Reply:
I know it can't be newer, older likely works. I'll update